### PR TITLE
Update admin-key-concepts.md

### PR DESCRIPTION
**Pull request by msbemba**
Changes to the FAQ answer On yammer basic users vs Enterprise

### CODE REVIEW

**Comment by msbemba** [comment]
@mypil Made the required changes and created this pull request.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by dianef77** [comment]
@IrenePasternack please polish and then merge.

File: yammer/get-started-with-yammer/admin-key-concepts.md [hunk 1]
```diff
@@ -124,7 +124,9 @@ A: It depends on whether you configure your Yammer network to enforce Office 365
     
 - If you do not choose to enforce Office 365 identity, users who are on the same email domain as your Yammer network but do not have Office 365 licenses can join the network by creating a Yammer account at http://www.yammer.com.
     
-    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users can be made admins, create groups, and use all Yammer Enterprise features. If they are made admins, they have to get to the Yammer admin center by clicking the **Settings** icon in Yammer, and selecting **Network Admin.**
+    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users need Enterprise license if they are added to the enterprise enabled network.
```
Comment (nenonix) [inline yammer/get-started-with-yammer/admin-key-concepts.md]:
```suggestion
    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users need a Yammer Enterprise license if they are added to the enterprise enabled network.
```

File: yammer/get-started-with-yammer/admin-key-concepts.md [hunk 2]
```diff
@@ -124,7 +124,9 @@ A: It depends on whether you configure your Yammer network to enforce Office 365
...
+
```
Comment (nenonix) [inline yammer/get-started-with-yammer/admin-key-concepts.md]:
```suggestion
```

**Comment by mypil** [comment]
@IrenePasternack  - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

**Comment by mypil** [comment]
@IrenePasternack - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

**Comment by mypil** [comment]
@IrenePasternack - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

**Comment by mypil** [comment]
@IrenePasternack - We would like to follow up on this.  Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

cc: @pamgreen

**Comment by mypil** [comment]
@IrenePasternack  - We would like to follow up. Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

**Comment by mypil** [comment]
@IrenePasternack @AndreaBarr - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #367?

Thank you.

cc: @pamgreen-msft

**Comment by AndreaBarr** [comment]
@mypil It looks like Irene is out of the office for a while so I sent an inquiry to her backups so we can get this approved for merging.

File: yammer/get-started-with-yammer/admin-key-concepts.md (hunk 1 above)
Comment (nenonix) [inline yammer/get-started-with-yammer/admin-key-concepts.md]: These users need a Yammer Enterprise license if they are added to the Enterprise enabled network.

File: yammer/get-started-with-yammer/admin-key-concepts.md [hunk 3]
```diff
@@ -124,7 +124,9 @@ A: It depends on whether you configure your Yammer network to enforce Office 365
     
 - If you do not choose to enforce Office 365 identity, users who are on the same email domain as your Yammer network but do not have Office 365 licenses can join the network by creating a Yammer account at http://www.yammer.com.
     
-    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users can be made admins, create groups, and use all Yammer Enterprise features. If they are made admins, they have to get to the Yammer admin center by clicking the **Settings** icon in Yammer, and selecting **Network Admin.**
+    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users need Enterprise license if they are added to the enterprise enabled network.
+    
+ - Users will need to have an enterprise license to use Yammer enterprise features.
```
Comment (nenonix) [inline yammer/get-started-with-yammer/admin-key-concepts.md]: Users will need to have an Enterprise license to use Yammer Enterprise features.

**Comment by nenonix** [comment]
@mypil @msbemba there is a problem with https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/tree/patch-8 (unknown repository)

**Comment by msbemba** [comment]
> @mypil @msbemba there is a problem with https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/tree/patch-8 (unknown repository)

I found the issue and created a new PR https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/pull/529 .

**Pull request closed without merging**